k2/python/k2/fsa.py

```diff
@@ -18,7 +19,48 @@
 
 class Fsa(object):
 
-    def __init__(self, s: str, openfst: bool = False):
+    def __init__(self, tensor: torch.Tensor,
```

Comment:
OK... I notice we only have an Fsa object and not FsaVec.  FsaVec is actually going to be the more common case.  For now I think we can have both (not for this PR!) but we may eventually decide to unify them.

Reply:
That's right, I planned to add FsaVec as a separate PR. But I'm not super clear yet whether or not I should add it as a separate class or just have one class. I'll create an issue for that and we can discuss from there. Once we have a conclusion, and can then implement it.

Reply:
Yeah it's not clear to me either.  I kind of assumed Fangjun would be doing this part (FsaVec).  I will follow up with some other comments on the design.  In fact I'll create an issue.

Reply:
I did not add `FsaVec` in Python because the construction of `FsaVec` had bugs which were fixed by #226 .